client/src/partials/reports/global_transaction/global_transaction.html

```diff
@@ -70,10 +64,9 @@
                 <td width="20%"> {{ record.trans_id }} </td>
```

Comment:
@lomamech , 20% + 20% + 20% + 20% + 20% + 50% = 150%.  Please correct your widths so that the text is visible!